Stefan Jones <cretin@gentoo.org>:
	Some more include dirs are needed in certain places when bootstraping
	a clean system to make g++

--- gcc-3.3.1/gcc/config/i386/t-cygming.jj	2003-10-01 21:26:15.614537232 +0100
+++ gcc-3.3.1/gcc/config/i386/t-cygming	2003-10-01 21:29:59.550493792 +0100
@@ -8,7 +8,8 @@
 # If we are building next to winsup, this will let us find the real
 # limits.h when building libgcc2.  Otherwise, winsup must be installed
 # first.
-LIBGCC2_INCLUDES = -I$(srcdir)/../winsup/w32api/include
+LIBGCC2_INCLUDES = -I$(srcdir)/../winsup/w32api/include \
+	-I$(srcdir)/../winsup/cygwin/include
 
 LIB2FUNCS_EXTRA = $(srcdir)/config/i386/w32-shared-ptr.c
 
@@ -19,3 +20,4 @@
 STMP_FIXPROTO=
 STMP_FIXINC=stmp-fixinc
 
+CRTSTUFF_CFLAGS += -isystem $(srcdir)/../winsup/cygwin/include
